### ISSUE TITLE
[Suggestion] LIFT Concat ENV()

### ISSUE THREAD

**Issue by johannesschobel** [prisma/lift#138]
Dear Prisma Team,

first of all - i really (!) love `Prisma` and especially the new `Lift`.

I would like to make a suggestion, as i just stumbled upon this problem - and was struggling to use respective feature.

In your `schema.prisma` file you specify your database connection string like this
```
url = "postgresql://postgres:postgres@localhost:5432/prisma?schema=prisma
```

Of course, this connection string contains sensitive information, like the password. So you can move this all in a typical `.env` file and use an `env` variable for this.
```
url = env("DB_CONNECTION")

# in your .env file
DB_CONNECTION="postgresql://postgres:postgres@localhost:5432/prisma?schema=prisma"
```

However, in my application, i already have some `.env` variables set up for
```
DB_USER
DB_PASSWORD
DB_NAME
DB_HOST
DB_PORT
DB_TYPE
```

So - in order to reduce duplicating those env variables, i would like to "concatenate" those variables within the connection string.
I thought about something like this:

```
url = env("DB_TYPE") + "://" . env("DB_USER") + ":" + env("DB_PASSWORD") + "@" + env("DB_HOST") + ":" + env("DB_PORT") + "/" + env("DB_NAME")
```

Basically, i would be building the connection string from the already pre-defined env variables. This throws, however, an `Unexpected Token` exception at the first `+` (plus sign).

What do you think about this suggestion? Would that be handy?
If you could point me to the right direction, maybe i can make a PR with this for you?

All the best and thanks for your time in reading this and helping me out!

**Comment by pantharshit00** [comment]
Hi,

Thanks for the kind words. You know you are not the only one requesting for this feature. This has been already requested here: https://github.com/prisma/photonjs/issues/206

So I will need to close this as a duplicate but you can add your +1 in that other issue.

**Comment by johannesschobel** [comment]
Dear @pantharshit00 ,
thanks for your response - unfortunately, i didn't find the other issue. I thought it was `Lift` related. I already left my `+1` ;)

thanks a lot for your time and help!
